information-schema/information-schema-processlist.md

```diff
@@ -85,6 +91,8 @@ RESOURCE_GROUP: default
 * `RESOURCE_GROUP` 列：显示对应的资源组名称。
 * `SESSION_ALIAS` 列：显示当前连接的别名。
 * `ROWS_AFFECTED` 列：显示语句当前影响的行数。
+* `TIDB_CPU` 列：显示语句当前占用 TiDB 服务器 CPU 时间，单位是秒。
```

Comment:
Maybe we can take an example to explain what does the cpu time means and how we calculate the cpu time.

Reply:
Explain it depends on topSQL feature to collect tidb cpu now.